homework-9/main.js

```diff
@@ -0,0 +1,68 @@
+// Bai 1: tinh thue va luong rong
+const getSalery = (luong) => {
+    if(luong >= 15000000){
+        console.log("Thuế thu nhập = ",luong*30/100);
```

Comment:
Để ý các dấu cách, dấu nháy
```suggestion
        console.log('Thuế thu nhập = ', salary * 30/100);
```